src/controllers/__tests__/ioFimsController.test.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  ResponseSuccessAccepted,
+  ResponseSuccessJson,
+} from "@pagopa/ts-commons/lib/responses";
+
+import mockReq from "../../__mocks__/request";
+import IoFimsController from "../fimsController";
+import { IoFimsAPIClient } from "../../clients/io-fims";
+
+import ApiClient from "../../services/apiClientFactory";
+import IoFimsService from "../../services/fimsService";
+import ProfileService from "../../services/profileService";
+
+import { ulid } from "ulid";
+
+import {
+  mockedUser,
+  mockedInitializedProfile,
+} from "../../__mocks__/user_mock";
+const API_KEY = "";
+const API_URL = "";
+const API_BASE_PATH = "";
+
+const mockGetAccessHistory = jest.fn();
+const mockRequestExport = jest.fn();
+
+jest.mock("../../services/fimsService", () => {
+  return {
+    default: jest.fn().mockImplementation(() => ({
+      getAccessHistory: mockGetAccessHistory,
+      requestExport: mockRequestExport,
+    })),
+  };
+});
+
+const mockGetProfile = jest.fn();
+
+jest.mock("../../services/profileService", () => {
+  return {
+    default: jest.fn().mockImplementation(() => ({
+      getProfile: mockGetProfile,
+    })),
+  };
+});
+
+const client = IoFimsAPIClient(API_KEY, API_URL, API_BASE_PATH);
+const apiClient = new ApiClient("XUZTCT88A51Y311X", "");
+
+const ioFimsService = new IoFimsService(client);
+const profileService = new ProfileService(apiClient);
+
+const mocks = {
+  accessHistoryPage: {
+    data: [],
+  },
+  exportRequest: {
+    id: ulid(),
+  },
+  user: mockedUser,
+  profile: mockedInitializedProfile,
+};
+
+describe("IoFimsController#getAccessHistory", () => {
```

Comment:
Can we add a test case to ensure that we are returning `422` instead of a `400`?

Reply:
not needed imho, now we are returning 400 (I aligned the specs)